nixos/foundation.tt

```diff
@@ -1,29 +1,47 @@
 [% WRAPPER layout.tt title="Stichting NixOS Foundation" menu='nixos' %]
 [% PROCESS common.tt %]
 
-<h2>General</h2>
+<div class="row-fluid">
+<div class="span6">
+<h2>About The Foundation</h2>
+
+<p>The mission of the foundation is to support the Nix ecosystem's
+infrastructure, and projects implementing the purely functional
+deployment model.</p>
+
+<p>In particular, the foundation supports Nix, Nixpkgs, NixOS, and
```

Comment:
-> `supports Nix, Nixpkgs and NixOS, and operates`.

Reply:
Fixed